cumulus/parachains/runtimes/assets/asset-hub-westend/tests/tests.rs

```diff
@@ -939,6 +950,87 @@ fn test_assets_balances_api_works() {
 		});
 }
 
+#[test]
+fn authorized_aliases_work() {
+	ExtBuilder::<Runtime>::default()
+		.with_tracing()
+		.with_collators(vec![AccountId::from(ALICE)])
+		.with_session_keys(vec![(
+			AccountId::from(ALICE),
+			AccountId::from(ALICE),
+			SessionKeys { aura: AuraId::from(sp_core::sr25519::Public::from_raw(ALICE)) },
+		)])
+		.build()
+		.execute_with(|| {
+			let alice: AccountId = ALICE.into();
+			let local_alice = Location::new(0, AccountId32 { network: Some(Westend), id: ALICE });
+			let alice_on_sibling_para =
+				Location::new(1, [Parachain(42), AccountId32 { network: None, id: ALICE }]);
+			let alice_on_relay = Location::new(1, AccountId32 { network: None, id: ALICE });
+			let bob_on_relay = Location::new(1, AccountId32 { network: None, id: [42_u8; 32] });
+
+			// neither `alice_on_sibling_para`, `alice_on_relay`, `bob_on_relay` are allowed to
+			// alias into `local_alice`
+			for aliaser in [&alice_on_sibling_para, &alice_on_relay, &bob_on_relay] {
+				assert!(!<XcmConfig as xcm_executor::Config>::Aliasers::contains(
+					aliaser,
+					&local_alice
+				));
+			}
+
+			// Alice explicitly authorizes `alice_on_sibling_para` to alias her local account
+			assert_ok!(PolkadotXcm::add_authorized_alias(
+				RuntimeHelper::origin_of(alice.clone()),
+				Box::new(alice_on_sibling_para.clone().into())
+			));
+
+			// `alice_on_sibling_para` now explicitly allowed to alias into `local_alice`
+			assert!(<XcmConfig as xcm_executor::Config>::Aliasers::contains(
+				&alice_on_sibling_para,
```

Comment:
iiuc, for actual XCM execution, the origin specified as `let alice_on_sibling_para = Location::new(1, [Parachain(42), AccountId32 { network: None, id: ALICE }]);` will work in the xcm-executor only if the XCM from sibling parachain `1, [Parachain(42)]` includes e.g. those two instructions bellow, right?
```
DescendOrigin(AccountId32 { network: None, id: ALICE }])
AliasOrigin(local_alice),
```

Reply:
That is correct.

One cool example is Alice on Para42 doing something on AH without having to transfer fees, from Para42, but instead use her local AH account:
```rust
// called by Alice on Para42
pallet_xcm::send(
	Location::new(1, Parachain(1000)),
	Xcm(vec![
		AliasOrigin(AliceOnAH),
		PayFees(Whatever),
		DoWhatever
	])
);
```

Reply:
nice, cool, that could allow even more use-cases. Adding this example to [TODOs](https://github.com/orgs/paritytech/projects/145/views/8?pane=issue&itemId=83905683), this could be a nice e2e test (chopsticks or emulated)